node/tests/TestUtilities.ts

```diff
@@ -387,6 +387,7 @@ export const getClientConfigurationOption = (
             port,
         })),
         protocol,
+        useTLS: parseCommandLineArgs()["tls"] == "true",
```

Comment:
ok - this will work - not sure why we don't check addresses here too, rather than getting command-line parameters in each test separately.  

Will clean this up in a follow-up PR